File: b_rank_skillcheck_archive/choco_sniper_easy_99.cs
// 実行時間 300ms
using static System.Console;
using static System.Linq.Enumerable;

internal class Program
{
    static void Main()
    {
        int o = 0;
        int[] n = ReadLine().Split().Select( int.Parse ).ToArray();

        if( n[0] != 0 ) {
            foreach( int _ in Range( 0, n[1] ) ) {
                o += new int[] { ReadLine().Split().Select( int.Parse ).Sum(), 0 }.Max();
            }
        }
        WriteLine( o );
        return;
    }
}

File: b_rank_skillcheck_archive/choco_sniper_easy_99.py
# 実行時間 170ms
import sys

r = sys.stdin.readline
o = 0
n, m = list(map(int, r().split()))

if n != 0:
    for _ in [0] * m:
        e = sum(list(map(int, r().split())))

        if 0 < e:
            o += e
print(o)
